oak/server/rust/oak_runtime/src/runtime/introspection_events.rs

```diff
@@ -44,9 +44,10 @@ impl Runtime {
             event_details: Some(event_details),
         };
 
-        // TODO(#913): Push the event over to a channel, to be consumed by the
-        // introspection aux server.
-        debug!("Introspection event recorded: {:?}", event);
+        let sender = self.introspection_event_sender.clone();
+        if let Err(error) = sender.send(event) {
+            warn!("Failed to send introspection event to channel: {:?}", error);
+        }
```

Comment:
Why is this cloning the sender for each new event that needs to be broacast?

Reply:
Also not needed, I think I carried over the cloning from another channel lib, which required it & didn't double check whether it was still needed.